RoddyCore/src/de/dkfz/roddy/config/loader/ConfigurationFactory.groovy

```diff
@@ -148,6 +148,8 @@ class ConfigurationFactory {
                         availableConfigurations[iccSub.id] = iccSub
                     }
 
+                } catch (UnknownConfigurationFileTypeException ex) {
```

Comment:
Yes, there are some predefined exception types. However I'll try to make sure to introduce only a small handful of custom exception types. But we can discuss the exception situation.